rllib/utils/files.py

```diff
@@ -0,0 +1,65 @@
+import os
+from pathlib import Path
+from typing import Dict
+
+
+def dir_contents_to_dict(dir: str) -> Dict:
```

Comment:
Some utilities. Only needed in our test cases for convenience!

Reply:
can you add a comment here about we only use this util for unit tests right now, and please be careful using it in production, since it's not really been battle tested.